### ISSUE TITLE
Is you maxpooling layer wrong?

### ISSUE THREAD

**Issue by eliottbrion** [orobix/retina-unet#43]
Hello,

I checked both shapes of conv1 and pool1 and they don't seem to be coherent:
K.int_shape(conv1)
(None, 32, 48, 48)
K.int_shape(pool1)
(None, 16, 24, 48)
It looks like you're doing the maxpooling operation on the feature map.

In retina-unet/src/retinaNN_training.py, don't you think that you should replace the 37th line
pool1 = MaxPooling2D((2, 2))(conv1)
by
pool1 = MaxPooling2D((2, 2),data_format='channels_first')(conv1)?

Eliott

**Comment by itachi1232gg** [comment]
@eliottbrion I think you are right, I found this error as well. What interesting is, it does not affect the results a lot.

**Comment by liuzhibo-delta** [comment]
I agree with you , the UpSampling Layer as well as the Maxpooling Layer are Wrong . Because of the parameter data_format='channels_first' is missing . You can correct it .
